plugins/modules/system/xfconf.py

```diff
@@ -225,6 +226,10 @@ def _get(self):
     def state_get(self):
         self.vars.value = self.vars.previous_value
         self.vars.previous_value = None
+        self.module.deprecate(
+            msg="State 'get' is deprecated. Please use the module xfconf_info instead.",
```

Comment:
Ansible adds a period at the end of the message.
```suggestion
            msg="State 'get' is deprecated. Please use the module xfconf_info instead",
```